src/cap-description.adoc

```diff
@@ -85,44 +85,81 @@ superfluous when the capability does not grant either <<r_perm>> or <<w_perm>>.
 Therefore, it is only possible to encode a subset of all combinations.
 
 .Permissions widths depending on XLENMAX
-[#perms_bit_width,options=header,align="center",width="55%"]
+[#perms_bit_width,options=header,align="center",width="75%",cols="^1,^3,5"]
 |==============================================================================
-^| XLENMAX ^| Permissions width
-^| 32      ^| {cap_rv32_perms_width}
-^| 64      ^| {cap_rv64_perms_width}
+^| XLENMAX ^| Permissions width      ^| Comment
+^| 32      ^| {cap_rv32_perms_width} ^| Encodes some combinations of 5 permission bits and the Mode bits
+^| 64      ^| {cap_rv64_perms_width} ^| Separate bits for each permission, and the Mode bit is not included
 |==============================================================================
 
+NOTE: While XLENMAX=32 currently uses the same number of bits for permission encodings, it also includes the mode bit of {cheri_mode_ext_name} which is separate for XLENMAX=64.
+
+The quadrant is taken from bits [4:3] of the permissions encoding. The meaning for bits [2:0]
+are shown in <<cap_perms_encoding32>> for each quadrant.
+
+Quadrants 2 and 3 are arranged to implicitly grant future permissions which may be added with the
+existing allocated encodings.  Quadrant 0 does the opposite - the encodings are allocated _not_
+to implictly add future permissions, and so granting future permissions will require new encodings.
+
 .Encoding of architectural permissions for XLENMAX=32
-[#cap_perms_encoding32,width="60%",options=header,cols="^2,^1,^1,^1,^1,^1",align="center"]
+[#cap_perms_encoding32,width="100%",options=header,cols="^2,^1,^1,^1,^1,^1,^2,4",align="center"]
 |==============================================================================
-| Encoding | R | W | C | X | ASR
-| 0b0000   |   |   |   |   |
-| 0b0001   5+| *reserved*
-| 0b0010   |   | ✔ |   |   |
-| 0b0011   |   | ✔ | ✔ |   |
-| 0b0100   | ✔ |   |   |   |
-| 0b0101   | ✔ |   | ✔ |   |
-| 0b0110   | ✔ | ✔ |   |   |
-| 0b0111   | ✔ | ✔ | ✔ |   |
-| 0b1000   | ✔ |   |   | ✔ |
-| 0b1001   | ✔ |   | ✔ | ✔ |
-| 0b1010   | ✔ | ✔ |   | ✔ |
-| 0b1011   | ✔ | ✔ | ✔ | ✔ |
-| 0b1100   | ✔ |   |   | ✔ | ✔
-| 0b1101   | ✔ |   | ✔ | ✔ | ✔
-| 0b1110   | ✔ | ✔ |   | ✔ | ✔
-| 0b1111   | ✔ | ✔ | ✔ | ✔ | ✔
+| Encoding[2:0] | R | W | C | X | ASR | Mode | Notes
+8+| *Quadrant 0: Non-capability data read/write*
```

Comment:
Nit: As we add non-memory permissions to this quad this description will become inaccurate.

Reply:
True, we will need to update it once that happens.